File: 167. Two Sum II - Input Array Is Sorted/Solution.py
class Solution:
    def twoSum(self, num: List[int], target: int) -> List[int]:
        l,r = 0,len(num)-1
        while l<r:
            curSum = num[l] + num[r] 
            if curSum == target:
                return l+1,r+1
            elif curSum > target:
                r-=1
            else:
                l+=1
        return []

        

File: 167. Two Sum II - Input Array Is Sorted/Solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& num, int target) {
        int l = 0,r = num.size()-1;
        while(l<r){
            int curSum = num[l]+num[r];
            if(curSum == target)
                return {l+1,r+1};
            else if(curSum > target)
                r--;
            else
                l++;
        }
        return {-1,-1};
    }   
};